src/inputs/internal/SelectFieldBase.tsx

```diff
@@ -45,6 +45,8 @@ export interface BeamSelectFieldBaseProps<O, V extends Value> extends BeamFocusa
   contrast?: boolean;
   /** Placeholder content */
   placeholder?: string;
+  /** Only used for Single Select Fields. If set, prepends an option with a `undefined` value at the top of the list */
```

Comment:
Thought: Should we exclude it from `MultiSelectField`'s props then?

Reply:
Makes sense to me.